src/PostgREST/Config.hs

```diff
@@ -88,18 +81,105 @@ corsPolicy req = case lookup "origin" headers of
 prettyVersion :: Text
 prettyVersion = intercalate "." $ map show $ versionBranch version
 
+-- | Default OS-specific path to config file
+defConf :: FilePath
+defConf =
+#ifdef mingw32_HOST_OS
+  "%PROGRAMDATA%\\postgrest\\postgrest.conf"
+#else
+  "/etc/postgrest.conf"
+#endif
+
 -- | Function to read and parse options from the command line
 readOptions :: IO AppConfig
-readOptions = customExecParser parserPrefs opts
-  where
-    opts = info (helper <*> argParser) $
-                    fullDesc
-                    <> progDesc (
-                    "PostgREST "
-                    <> toS prettyVersion
-                    <> " / create a REST API to an existing Postgres database"
-                    )
-    parserPrefs = prefs showHelpOnError
+readOptions = do
+  args <- customExecParser parserPrefs opts
+
+  when (caExample args) $ do
```

Comment:
Configurator can check whether a mandatory key is missing from the config or whether a value has the wrong type. It doesn't say which line is wrong but the key name should be enough information to debug the issue. For instance when I use an integer value for `db-anon-role` it warns me:

```
$ stack exec -- postgrest -c test.conf
Required config parameter "db-anon-role" is missing or of wrong type.
Try the --example-config option to see how to configure PostgREST.
```